packages/components/calendar/src/calendar.vue

```diff
@@ -75,11 +76,16 @@ const {
   pickDay,
   realSelectedDay,
   selectDate,
-  t,
-  i18nDate,
   validatedRange,
 } = useCalendar(props, emit, COMPONENT_NAME)
 
+const { t } = useLocale()
+
+const i18nDate = computed(() => {
+  const pickedMonth = `el.datepicker.month${date.value.format('M')}`
+  return `${date.value.year()} ${t('el.datepicker.year')} ${t(pickedMonth)}`
+})
+
```

Comment:
Question

Why is that logic separated from hooks? Curious about the standards!

Reply:
@ryuhangyeong the reason I extracted this out of the hook is that this computed member is only used for the template rendering, for instance styles, classes getters should also be extracted from the hooks.

```ts
const style = computed(() => ({ someStyle: 'value' }))
const kls = computed(() => (['kls1', { kls2: true }]))
```

Reply:
@jw-foss  Thanks~ I will refer to your opinion.